charts/apisix-ingress-controller/Chart.yaml

```diff
@@ -15,13 +15,14 @@
 # limitations under the License.
 #
 apiVersion: v2
-name: ingress-apisix
+name: apisix-ingress-controller
 description: Apache APISIX Ingress Controller for Kubernetes
 icon: https://apache.org/logos/res/apisix/apisix.png
 keywords:
   - ingress
   - apisix
   - nginx
+  - crd
 type: application
 version: 0.1.0
```

Comment:
Since 0.2.0 was released, we can change the `version` to `0.2.0`.

Reply:
version? or appVersion?

Reply:
I think you mean `appVersion`, change the `appVersion` to `0.2.0` is better, also need to be modified `image.tag` in values, but `apisix-ingress-controller` has not yet been release 0.2.0 docker image.

Reply:
Yes，the appVersion.

Reply:
> Yes，i mean the appVersion.

The above explains why not change the `appVersion` to 0.2.0, Can help publish 0.2.0 docker image?

Reply:
Well, it needs the apache account.

Reply:
Just let it go right now. After the 0.2.0 is released, let's change it.